### ISSUE TITLE
ImportError: DLL load failed while importing_simbody: The specified model could not be found.

### ISSUE THREAD

**Issue by doba-95** [opensim-org/opensim-core#3213]
Hello everyone,

I am struggling now for the past 7 hours with setting up my OpenSim Python interface. I followed the steps from Scripting in Python under the section __Step-by-step instructions when using Anaconda with OpenSim 4.3+__, but I keep receiving the Following Error:

`(opensim_scripting) C:\OpenSim 4.3\sdk\Python>python
Python 3.8.13 (default, Mar 28 2022, 06:59:08) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import opensim as osim
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\OpenSim 4.3\sdk\Python\opensim\__init__.py", line 5, in <module>
    from .simbody import *
  File "C:\OpenSim 4.3\sdk\Python\opensim\simbody.py", line 13, in <module>
    from . import _simbody
ImportError: DLL load failed while importing _simbody: The specified module could not be found.`

What I have done so far: 
- uninstalled my previous Python 3.10 
- reinstalled anaconda and follwed the steps again
- checked whether the module is in the opensim folder 
- created a PYTHONPATH
- used python -m pip isntall . --use-feature=in-tree-build

I am using Win 11 Pro and the Python Version 3.8.13 from the environment file provided in the guidline.
Help is much appreciated! 
Cheers

**Comment by aymanhab** [comment]
@doba-95 While we continue to support the old installation mechanism, we're moving into self-contained conda packages for OpenSim here https://anaconda.org/opensim-org/opensim please give that a try first. My recollection is that the PATH needed to include the sdk folder containing simbody's dlls as well which is not on the above list of steps. Please let us now at your earliest convenience if you run into issues with the conda pathway.

**Comment by doba-95** [comment]
@aymanhab Thanks for your response and sorry for my late answer. I uninstalled opensim 4.3 and installed 4.4 beta. I uninstalled anaconda, reinstalled it and installed the opensim package you mentioned. I also added the the C:\OpenSim 4.4\sdk\Python\opensim folder ( I also tried it with the path where the package in Anaconda is located \Anaconda3\lib\opensim) to the PATH in the environment variables where the dlls are contained (I hope this is what you meant) and changed the path in the __init__.py file to the same folder. 
Nevertheless, I keep receiving the same error message: 
`Traceback (most recent call last):

  File ~\.spyder-py3\temp.py:13 in <module>
    import opensim as osim

  File ~\Anaconda3\lib\opensim\__init__.py:9 in <module>
    from .simbody import *

  File ~\Anaconda3\lib\opensim\simbody.py:13 in <module>
    from . import _simbody

ImportError: DLL load failed while importing _simbody: The specified module could not be found.`

Is it a possibility that the endings within the folder of the pyd-files play a role? I double checked the sdk folder of a fellow worker and his files were named e.g: _simbody.pyd (type: pyd file) while mine is called _simbody (type: pyd file)?
Once I edited the ending (to .pyd): the error message changed to: 
 `File ~\.spyder-py3\temp.py:13 in <module>
    import opensim as osim

  File ~\Anaconda3\lib\opensim\__init__.py:9 in <module>
    from .simbody import *

  File ~\Anaconda3\lib\opensim\simbody.py:13 in <module>
    from . import _simbody

ImportError: cannot import name '_simbody' from partially initialized module 'opensim' (most likely due to a circular import) (C:\Users\dominicbachmann\Anaconda3\lib\opensim\__init__.py)`

**Comment by doba-95** [comment]
I solved the issue! 
In the beginning  I simply followed the link and used `conda install -c opensim-org opensim`. Today I realized that the package installed with this is called opensim-4.4-py37np120 ... which was not matching my python version! 
I now downloaded the file opensim-4.4-py39np120.taz.bz2 and installed it through 
`conda install C:\ (directory where the package is located)\opensim-4.4-py39np120.tar.bz2`, added the DLL-Path to opensim located in the anaconda3\lib\opensim folder. 

I close this issue now.
